apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1084,10 +1084,21 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                             t("console:develop.features.applications.forms.inboundOIDC.sections" +
                                 ".accessToken.fields.bindingType.label")
                         }
-                        hint={ t("console:develop.features.applications.forms.inboundOIDC.sections" +
-                            ".accessToken.fields.bindingType.description", {
-                            productName: config.ui.productName
-                        }) }
+                        hint={ 
+                            <Trans values={{productName: config.ui.productName}}
+                                i18nKey={
+                                    "console:develop.features.applications.forms.inboundOIDC.sections" +
+                                    ".accessToken.fields.bindingType.description"
+                                }
+                            >
+                                productName can bind the 
+                                <code><b>access_token</b></code> 
```

Comment:
There is a reusable component for this. Better to use that.

```tsx
import { Code } from "@wso2is/react-components";

<Code>access_token</Code> 
```

Reply:
Fixed using ```<Code withBackground>```